netutils/ip.py

```diff
@@ -596,3 +597,42 @@ def get_usable_range(ip_network: str) -> str:
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def sort_list_ips(ips: t.Union[str, t.List[str]], sort_type: str = "network") -> str:
```

Comment:
I thought I hit send, but I think we should

* change the name, thinking either ip_sorted or ips_sorted
* This should return a list, imho.

Reply:
Changing the name of the function to sort_ips makes sense IMO as that's what's being done, but returning a list of strings I don't think makes sense as the use cases I have for this are taking a list of strings, ie from a config context YAML and I'd want the string form of them for the generated template or for python where the same is being done. Returning a list of strings would require additional work in the template to concatenate the strings together.

Reply:
* the name follows the pattern in the rest of the file, i don't see why this would be different
* `Returning a list of strings would require additional work in the template to concatenate the strings together.` doing a `{{ (vals | ip_sorted).join(',') }}` seems like a reasonable trade off to keeping it a data structure vs an opinionated concatenation, as the next time you may want to do `{{ (vals | ip_sorted).join(';') }}` and would have to split and then rejoin.

Reply:
I've renamed the function to `get_ips_sorted` to match the pattern of the other functions. I've also updated it to return a list instead of a concatenated string.